kickoff/views/releases.py

```diff
@@ -81,6 +81,8 @@ def post(self, releaseName):
         if form.enUSPlatforms.data:
             log.debug('%s: enUSPlatforms being changed to: %s', releaseName, form.enUSPlatforms.data)
             release.enUSPlatforms = form.enUSPlatforms.data
+        log.debug('%s: shippedAt being changed to: %s', releaseName, form.shippedAt.data)
+        release.shippedAt = form.shippedAt.data
```

Comment:
This field is optional in the form, but here we don't check its value. Is this expected?